wagtail/contrib/typed_table_block/static_src/typed_table_block/scss/typed_table_block.scss

```
@@ -42,6 +48,10 @@
 
         td {
             vertical-align: top;
+            min-width: 320px;
+            &:last-child {
+                min-width: 5em;
```

Instead of setting this to a value that might be excessive if a user increases font size, we could just unset the max-width:

```suggestion
                min-width: initial;
```

---

I did this to prevent the button going under the scroll bar. I also used `td:not(:last-child)`, which has the same results.